services/views.py

```diff
@@ -0,0 +1,49 @@
+import json
+import re
+from django.http             import JsonResponse
+from django.views            import View
+
+from services.models          import Service, Category
+
+class CategoryView(View):
+    def get(self, request):
+        try:
+            all_categories = Category.objects.all()
+            category_list = [{
+                            'state_code'     :category.id,
+                            'name'           :category.name,
+                            'icon_image_url' :category.icon_image_url,
+                        } for category in all_categories]
+            
+            return JsonResponse({'RESULT':category_list}, status=200)
+
+        except KeyError:
+            return JsonResponse({'MESSAGE': 'KEY ERROR OCCURED!'},status=400)
```

Comment:
```suggestion
            return JsonResponse({'MESSAGE': 'KEY_ERROR_OCCURED!'},status=400)
```
status 메세지에서 띄어쓰기는 보통 underbar로 구분합니다 :) 파일 안에 있는 모든 메세지 띄어쓰기 한 번 수정해주세요 :)
test code도 같이 수정부탁드립니다!

Reply:
underbar 메모하겠습니다!